spec/models/miq_group_spec.rb

```diff
@@ -268,7 +268,6 @@
       gc = FactoryGirl.create(:miq_group, :description => 'C', :tenant => tenant)
       ga = FactoryGirl.create(:miq_group, :description => 'a', :tenant => tenant)
       gb = FactoryGirl.create(:miq_group, :description => 'B', :tenant => tenant)
-      FactoryGirl.create(:miq_group, :description => 'X')
```

Comment:
This test was assuming that creating a group without a tenant would cause that group to not have a tenant... this assumption is now incorrect with this PR so this line has been removed.